src/core/client/admin/components/ModerateCard/ReactionDetailsContainer.tsx

```diff
@@ -0,0 +1,136 @@
+import { Localized } from "@fluent/react/compat";
+import React, { FunctionComponent } from "react";
+import { graphql, RelayPaginationProp } from "react-relay";
+
+import NotAvailable from "coral-admin/components/NotAvailable";
+import {
+  useLoadMore,
+  withPaginationContainer,
+} from "coral-framework/lib/relay";
+import { BaseButton, Flex } from "coral-ui/components/v2";
+import { Button } from "coral-ui/components/v3";
+
+import { ReactionDetailsContainer_comment } from "coral-admin/__generated__/ReactionDetailsContainer_comment.graphql";
+import { ReactionDetailsContainerPaginationQuery } from "coral-admin/__generated__/ReactionDetailsContainerPaginationQuery.graphql";
+
+import styles from "./ReactionDetailsContainer.css";
+
+interface Props {
+  comment: ReactionDetailsContainer_comment;
+  relay: RelayPaginationProp;
+  onUsernameClick: (id?: string) => void;
+}
+
+const ReactionDetailsContainer: FunctionComponent<Props> = ({
+  comment,
+  relay,
+  onUsernameClick,
+}) => {
+  const reactions = comment.reactions
+    ? comment.reactions.edges.map((edge) => edge.node)
+    : [];
+  const hasMore = relay.hasMore();
+  const [loadMore, isLoadingMore] = useLoadMore(relay, 20);
+
+  return (
+    <>
+      {reactions.map(
+        (reaction) =>
+          reaction.id && (
+            <Flex key={reaction.id}>
+              <BaseButton
+                onClick={() => {
+                  if (reaction.reacter?.userID) {
+                    onUsernameClick(reaction.reacter?.userID);
+                  }
+                }}
+                className={styles.button}
+              >
+                <span className={styles.username}>
+                  {reaction.reacter?.username ? (
+                    reaction.reacter.username
+                  ) : (
+                    <NotAvailable />
+                  )}
+                </span>
+              </BaseButton>
+            </Flex>
+          )
+      )}
+      {hasMore && (
+        <Localized id="moderateCardDetails-tab-reactions-loadMore">
+          <Button
+            variant="outlined"
+            color="secondary"
+            fontSize="extraSmall"
+            onClick={loadMore}
```

Comment:
For safeties sake and to prevent duplicate clicks on the load more button, would you mind encapsulating this in a callback similar to the following?

```
const onLoadMore = useCallback(() => {
    if (!relay.hasMore() || relay.isLoading()) {
      return;
    }

    loadMore();
  }, [loadMore]);
```

I don't think duplicate clicks ever caused any harm, but I've seen this done elsewhere in places.

Reply:
I think this is already taken care of by disabling the button with `disabled={isLoadingMore}` if it's loading more and only showing the button if it has more?